README.md

```diff
@@ -608,7 +606,7 @@ gpg --keyserver pool.sks-keyservers.net --recv-keys DD8F2338BAE7501E3DD5AC78C273
 See the section above on [Verifying Binaries](#verifying-binaries) for how to
 use these keys to verify a downloaded file.
 
-Previous releases may also have been signed with one of the following GPG keys:
+Other keys used to sign some previous releases include:
```

Comment:
Re: `include` … we should be clear on whether this is an exhaustive list or not. The previous wording sounded to me like it is?

Reply:
Yes, it's an exhaustive list.

Reply:
Maybe I can remove `include`?: `Other keys used to sign some previous releases:`